images/win/scripts/Installers/Install-SQLExpress.ps1

```diff
@@ -0,0 +1,76 @@
+################################################################################
+##  File:  Install-SQLExpress.ps1
+##  Desc:  Install SQL Express for Windows
+################################################################################
+function Start-Task {
+    param(
+        [String]$InstallPath,
+        [String]$Arguments,
+        [String]$SuccessMessage
+    )
+    $process = Start-Process -FilePath $InstallPath -ArgumentList $Arguments -Wait -PassThru
+    $exitCode = $process.ExitCode
+# Exit code -2067529716 is added since SQL Unpack procedure returns it on success.
+    if ($exitCode -eq 0 -or $exitCode -eq -2067529716)
+    {
+      Write-Host -Object "$SuccessMessage : ExitCode: $exitCode"
+    }
+    else
+    {
+      Write-Host -Object "Non zero exit code returned by the installation process : $exitCode."
+      exit $exitCode
+    }
+}
```

Comment:
I think that we should implement a few functions instead of this generic.
Something like `Download-FullSQLPackage`, `Unpack-SQLInstaller`, `Start-Installer` and we will be able to include arguments arguments inside function.

I don't like the current approach because: 
1) We allow `-2067529716` exit code for all processes but it is valid exit code only for unpack
2) we pass success message to this function.

Let's split to the few functions so we will put run arguments to these function too.